collectors/python.d.plugin/isc_dhcpd/README.md

```diff
@@ -41,11 +42,13 @@ Sample:
 
 ```yaml
 local:
-  leases_path       : '/var/lib/dhcp/dhcpd.leases'
-  pools       : '192.168.3.0/24 192.168.4.0/24 192.168.5.0/24'
```

Comment:
I have updated the configuration since seeing the secript this will never pass the check.
https://github.com/vsc55/netdata/blob/38ef84cbbc396cc3398e8ce08a96cc83efb6f717/collectors/python.d.plugin/isc_dhcpd/isc_dhcpd.chart.py#L146